pkg/download/id_extraction/id_extraction.go

```diff
@@ -95,9 +95,23 @@ func ExtractIDsIntoYAML(configsPerType project.ConfigsPerType) (project.ConfigsP
 	return configsPerType, nil
 }
 
+// invalidMeId finds ME IDs in the form of `nABC`, `rABC`, and `tABC`. They are most commonly mistakenly created by `\nABC`.
+var invalidMeId = regexp.MustCompile("[nrt][A-Z]+")
```

Comment:
Why is this defined like this when the ME ID is a much more general `[a-zA-Z_]+-[A-Fa-f0-9]{16}`?

Reply:
e.g. `nmz-1234567890ABCDEF` from `\nmz-1234567890ABCDEF` would match `meIDRegexPattern` but not `invalidMeId`.

Reply:
TBH, I've never seen a lowercase ME ID, and I think settings even does not allow it.
The sanity check is exactly having a lowercase `n` followed by uppercase characters. If there are only lowercase characters, how can we say that it's not part of the ME ID?

Maybe we need to challenge our assumption that ME IDs can be lowercase, and that would fix it as well.

Reply:
OK, that makes sense. The extra check handles exactly this case.